.gitlab/functional_test_sysprobe/system_probe.yml

```diff
@@ -51,7 +51,7 @@ kitchen_test_dummy_job_tmp:
   needs: []
   tags: ["arch:amd64"]
   rules:
-    !reference [ .on_system_probe_changes_or_manual ]
+    !reference [ .on_system_probe_or_e2e_changes_or_manual ]
```

Comment:
Do we want kitchen tests to run if e2e changes? It doesn't use e2e.

Reply:
This is what the rule is doing: https://github.com/DataDog/datadog-agent/blob/pducolin/APL-2196-run-test-infra-tests-on-ci/.gitlab-ci.yml#L1002-L1006

You might want to change the rule to isolate `e2e` changes